py/private/py_pytest_main.bzl

```diff
@@ -53,13 +53,15 @@ _py_pytest_main = rule(
     },
 )
 
-def py_pytest_main(name, py_library = default_py_library, deps = [], **kwargs):
+def py_pytest_main(name, py_library = default_py_library, deps = [], data = [], testonly = False, **kwargs):
```

Comment:
this is a test rule, when would you use testonly = False? i.e. should the default be True, or even force to True?

Reply:
Same reasoning as above. Changed to `True` 😄